mappings/services.json

```diff
@@ -1,4 +1,27 @@
 {
+  "settings": {
+    "index": {
+      "number_of_shards": 1
```

Comment:
We do, however we are going to move this down to 1 shard given our indices are so small and there will be an (admittedly small) efficiency bump by turning off sharding.